geoserver_init/wait-for.sh

```diff
@@ -137,7 +156,7 @@ if ! [ "$TIMEOUT" -ge 0 ] 2>/dev/null; then
   usage 3
 fi
 
-if [ "$HOST" = "" -o "$PORT" = "" ]; then
+if [ "$GS_HOST" = "" -o "$GS_PORT" = ""]; then
```

Comment:
Edit: Glitch in spellcheck. Now with sugesstion in the right place.
According to my IDE, the space should stay. Change at your discretion. 
Other than that everything looks fine. Init + wait-for-it.sh worked despite this.
Thank you, again.

```suggestion
if [ "$GS_HOST" = "" -o "$GS_PORT" = "" ]; then
```

Reply:
Done by rebasing.